k8s.io/deployment.yaml

```diff
@@ -31,7 +35,7 @@ spec:
           # Map all keys to files.
       containers:
       - name: nginx
-        image: nginx:1.10-alpine  # = :stable-alpine as of 8/23/2016
+        image: nginx:1.19-alpine  # = :stable-alpine as of 11/17/2020
```

Comment:
is this already running or is it a change?

Reply:
A change not tested in the canary namespace.

Reply:
Change reverted. I'll introduce the change in a followup PR.